changelog/issue-7443.md

```diff
@@ -0,0 +1,5 @@
+audience: worker-deployers
+level: minor
+reference: issue 7443
+---
+<!-- replace this text with your changelog entry.  See dev-docs/best-practices/changelog.md for help writing changelog entries. -->
```

Comment:
Can you add a description here?

Reply:
haha, no.. you were too fast to review, thanks
i was struggling with my shell setup, created several changelog entries, and when noticed, removed the ones that contained valid input .. 
should be fine now